File: 169. Majority Element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        a,b=0,0
        for i in nums:
            if b==0:
                a=i
                b+=1
            elif i==a:
                b+=1
            else:
                b-=1
        return a

File: 169. Majority Element.java
class Solution {
    public int majorityElement(int[] nums) {
        int a=0;
        int b=0;
        for(int i:nums){
            if(b==0){
                a=i;
                b+=1;
            }else if(i==a){
                b+=1;
            }else{
                b-=1;
            }
        }
        return a;
        
    }
}
